packages/truffle-contract/typings/index.d.ts

```diff
@@ -0,0 +1,11 @@
+declare module "truffle-contract" {
+  namespace TruffleContract {
+    export interface TruffleContract {
+      contractName: string;
+      contract_name: string;
```

Comment:
blech, both!?

Reply:
yuppp, I believe we currently support both.

Reply:
_sigh_ ok